tutorials/customizing-base-map-styles/index.md

```diff
@@ -0,0 +1,379 @@
+---
+title: Customizing base map styles
+description: If you are getting bored of looking at the same two default basemaps in Earth Engine, or need to use a specific visualization for functionality/convenience, this tutorial will show you how to edit all of its characteristics.
```

Comment:
This language/tone seems overly prescriptive. I'd suggest making the description more solution-focused, e.g.:
> This tutorial shows how to change the base map visualization in Earth Engine, which is useful for...

Reply:
Done.